### PR TITLE
Simplify DAA chapter

### PR DESCRIPTION
**Pull request by RiboRings**
Hi! This PR is meant to streamline the DAA chapter and make the code easy to execute and read for future tutorials.

### CODE REVIEW

**Comment by RiboRings** [comment]
Other things that could be considered:

- Change list of challenges and approaches into paragraphs and sections in a more narrative way
- Move list of DAA tools to end of chapter (additional resources)
- Agglomerate and subset only once (right now it is done twice from lines 175 and 344 on)
- Review text and consider section reorganisation

**Comment by antagomir** [comment]
Could you do at least those suggestions that are easy to immediately add on this PR?

File: 30_differential_abundance.Rmd [hunk 1]
```diff
@@ -339,26 +340,29 @@ that we specify.
...
-  p_adj_method = "fdr", 
-  prv_cut = 0,
-  group = "Geographical_location", 
-  struc_zero = TRUE, 
-  neg_lb = TRUE,
-  global = TRUE # multi group comparison will be deactivated automatically 
-)
+# Run ANCOM-BC at the genus level and only including the prevalent genera
+out <- ancombc2(data = altExp(tse, "genus")[prevalent.genera, ],
+                assay_name = "counts",
```
Comment (antagomir) [inline 30_differential_abundance.Rmd:357]: check if `assay.type` works as an argument name with the latest ANCOMBC version from Github?

**Review by antagomir — COMMENTED** [review]
You could also see if we need this many DAA methods in this chapter. Some of them could be added as supplementary material behind a link, unless the method has clear advantage compared to the other methods in some sense.

File: 30_differential_abundance.Rmd [hunk 2]
```diff
@@ -180,17 +172,23 @@ Let us here do aggregation to genus level, add relative abundance
...
+                       method = "relabundance")
+
+# Subset based on the relative abundance assay		       
+tse <- subsetByPrevalentFeatures(tse,
+                                 detection = 0,
+                                 prevalence = 10 / 100,
+                                 assay.type = "relabundance")
+
+# Add clr abundances
+tse <- transformCounts(tse,
```
[the remark this answers is not in the document]
Comment (RiboRings) [reply 30_differential_abundance.Rmd:189]: Ok! What about `mergeFeaturesByRank`? Is that a new version of `agglomerateByRank` or should I replace it with `agglomerateByRank`?

**Comment by RiboRings** [comment]
What do you think? Would it be an option to switch to Tengeler2020 for the DA chapter? It could be a good choice considering that it has a defined treatment/control design and a relatively small number of features, as opposed to peerj13075 which is currently used.

File: 30_differential_abundance.Rmd (hunk 1 above)
Comment (RiboRings) [reply 30_differential_abundance.Rmd:357]: Apparently `assay.type` is not yet supported by ANCOMBC (GitHub version).

File: 30_differential_abundance.Rmd (hunk 2 above)
[the remark this answers is not in the document]
Comment (antagomir) [reply 30_differential_abundance.Rmd]: Yes these are just the new names.

File: 30_differential_abundance.Rmd (hunk 1 above)
Comment (antagomir) [reply 30_differential_abundance.Rmd:307]: Yeah. Stilll in the making.

File: 20_beta_diversity.Rmd [hunk 3]
```diff
@@ -40,12 +40,12 @@ similarities in an optimal way, which is defined in different ways by different
...
 
 Based on the type of algorithm, ordination methods in microbiome research can
-be generally divided in two categories: unsupervised and supervized ordination.
+be generally divided in two categories: unsupervised and supervised ordination.
 The former includes Principal Coordinate Analysis (PCoA), Principal Component
 Analysis (PCA) and Uniform Manifold Approximation and Projection for Dimension
 Reduction (UMAP), whereas the latter is mainly represented by distance-based
-Redundance Analysis (dbRDA). We will first discuss unsupervized ordination
-methods and then proceed to supervized ones.
+Redundance Analysis (dbRDA). We will first discuss unsupervised ordination
```
Comment (antagomir) [inline 20_beta_diversity.Rmd:47]: it is Redundancy, not Redundance. Fix here and elsehwere?

File: book.bib [hunk 4]
```diff
@@ -1424,4 +1424,11 @@ @article{Tengeler2020
...
 
-
+@article{Schloss2023,
+  title={Rarefaction is currently the best approach to control for uneven sequencing effort in amplicon sequence analyses},
+  author={Schloss, Patrick D},
+  journal={bioRxiv},
+  pages={2023--06},
+  year={2023},
+  publisher={Cold Spring Harbor Laboratory}
+}
```
Comment (antagomir) [inline book.bib:1434]: add DOI?

**Review by antagomir — APPROVED** [review]
Merge when ready

**Pull request merged**